app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<section class="landing-page">
+ <div class="landing-cont">
+  <h1> BudgetBuddy</h1>
+    <div class="links-btns">     
+       <button class="sign-in-btn"><%= link_to "LOG IN", new_user_session_path %></button>
+       <button class="sign-up-btn"><%= link_to "SIGN UP", new_user_registration_path %></button>
+    </div>
+ </div
+</section>
```

Comment:
- Hey, please, kindly check your devise setup as a user cannot access your app after they signup, they get a message about confirmation from the UI, but they never get any confirmation message to their email and as a result, they can't log in to your app. please, can your kindly correct this issue so that your project can be well reviewed based on the changes requested by @Omar-Muhamad

*see the message*
![image](https://github.com/cosywasswa/Budget-Buddy/assets/59397159/73766f66-7f50-4342-81ce-5b46eb1db69a)

Reply:
Hi @zieeco and next reviewer, the confirmation link is accessible in the log folder inside development.log file.
Here is a video of this process. 
https://drive.google.com/file/d/1geBTIZG8umhdkXBVm0YiWa1tWRH2BU4a/view?usp=sharing
 U can also kindly contact me. Thank you